HW_2/task_5.py

```diff
@@ -0,0 +1,20 @@
+# 5 – Реализуйте алгоритм перемешивания списка
+
+from random import Random, randint
+
+N = int(input('Введите число '))
+numbers = []
+for i in range(N):
+    numbers.append(randint(-N,N+1))
+print(numbers)
+
+def mix(numbers):
+    list = numbers[:]
+    numbers_length = len(list)
+    for i in range(numbers_length):
+        index = randint(0, numbers_length - 1)
+        temp = list[i]
+        list[i] = list[index]
+        list[index] = temp
```

Comment:
```suggestion
        list[i], list[index] = list[index], list[i]
```
Это же Python, пользуйтесь преимуществами этого языка. Ни в каком в другом языке у вас так сделать не получится.